services/keycloak-dashboard.adoc

```diff
@@ -0,0 +1,79 @@
+= Keycloak dashboard in Grafana
+
+== Prerequesites
+
+* OpenShift Origin >= 3.7
+** https://docs.openshift.org/latest/install_config/index.html
+* OpenShift Ansible Broker
+** https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-openshift-ansible-broker
+* Metrics service deployment from the Service Catalog
+** https://github.com/aerogearcatalog/metrics-apb
+* Keycloak service deployment from the Service Catalog
+** https://github.com/aerogearcatalog/keycloak-apb
+
+*Note*
+
+Metrics service and Keycloak service must be provisioned within the same project,
```

Comment:
project is Openshift specific, should we use `namespace` instead?

Reply:
good point, will fix this